vars/documentationChecker.groovy

```diff
@@ -0,0 +1,51 @@
+
+
+// Add the following code to any Jenkinsfile.
+// It will return with success if there are no changes in the specified docs dir
+// and Will return with success if run on a non Pull Request branch
+
+// documentationChecker("docs")
+
+// docsDir ~~ the subdirectory within your repo that the documentation resides
+def documentationChecker(docsDir) {
+  if (env.CHANGE_ID != null) {
+    stage "docs PR checker"
+
+    tokens = "${env.JOB_NAME}".tokenize('/')
+    org = tokens[0]
+    repo = tokens[1]
+    branch = tokens[2]
+    sha = getOutput("git rev-parse").trim()
```

Comment:
oh actually i just went back to a jenkinsfile from a while ago, and apparently i made a function for exactly this. [`gitCommit()`](https://github.com/docker/jenkins-pipeline-scripts/blob/master/vars/gitCommit.groovy) will give you the sha.